src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -1013,9 +1013,9 @@ private void ResetWindowState()
             suppress_resize--;
         }
 
-        private void GrabCursor(bool grab)
+        private void GrabCursor(bool shouldGrab)
```

Comment:
I'm absolutely down with having a single method, but it needs a better name. GrabCursor implies that it should grab the cursor, not operate on the input parameter and grab/ungrab. If it had been a property, the name would have been fine, but not as a method.